serviceinsight/upgrades/index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Upgrade Guide
+reviewed: 2022-08-04
+related:
+---
```

Comment:
Ah, I get it. The engine is forcing you to have an index.md in the directory in order to have support-policy.md. But this is not that complex and you don't need that.

Here (and for Pulse as well) I would either:

1. Put all the support-policy.md info here, and make the title of this file "ServiceInsight support policy"
2. Don't create this directory at all, and create /serviceinsight/support-policy.md. It doesn't need to match the structure of NServiceBus and ServiceControl because it's not nearly as complex.

Reply:
@DavidBoike yes, exactly right. The engine requires a placeholder (enforces review tag as well). I think your suggestion works as well, especially since we're not going to have upgrade guides for ServiceInsight (and probably Pulse).

Reply:
Have gone with the suggested option 2